### ISSUE TITLE
return plot on search

### ISSUE THREAD

**Issue by jjtbsomhorst** [omdbapi/OMDb-API#195]
Hi,
currently when using the search api we don't get a plot returned. Would it be possible to implement this for searches?

**Comment by danieledwardharrington** [comment]
Yeah I feel like the limited fields we get in the return is a little disappointing. I'd like for just the whole object to get returned.
